pkg/web/utilities.go

```diff
@@ -0,0 +1,72 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/trisacrypto/envoy/pkg"
+	"github.com/trisacrypto/envoy/pkg/web/api/v1"
+	"github.com/trisacrypto/trisa/pkg/openvasp/traddr"
+)
+
+func (s *Server) TravelAddressUtility(c *gin.Context) {
```

Comment:
Should this be moved to `pages.go` for consistency?

Reply:
Moved!